src/data.js

```diff
@@ -803,3 +803,110 @@ export const disciplines = {
     },
   ],
 };
+export const vitals = [
+  {
+    id: "Health",
+    style: "healthPt",
+    scale: 15,
+    depends: ['Stamina'],
+    value: 3,
+    defaultValue: 3,
+    description:"Your Health; usually equals stamina + 3",
+  },
+  {
+    id: "Willpower",
+    style: "willPt",
+    scale: 15,
+    depends: ['Composure', 'Resolve'],
+    value: 0,
+    defaultValue: 0,
+    description:"Your Willpower; usually equals composure + resolve",
+  },
+  {
+    id: "Humanity",
+    style: "humanityPt",
+    scale: 10,
+    value: 7,
+    defaultValue: 7, 
+    description:"Your Humanity, a measure of resistance against the beast",
+  },
+  {
+    id: "Hunger",
```

Comment:
Hunger sa pri generovani postavy nikdy nemeni. To sa pouziva az v hre.

Reply:
Imho to tu vobec nemusi byt

Reply:
a nebude potom divne ked v hre pribudne polozka?
ad 2) ancillae zacinaju s hunger 2 alebo 3, nemal by o tom user vediet?

Reply:
aha... tak teraz som sa pozeral a som to nenasiel ci to ovplinuje hunger, len humanity jak si z nejakej nizsej generacie...
stejne si myslim ze to bude divne

Reply:
ja si myslim, ze Hunger pri generovani postavy nepotrebujes. Je to dolezite pro hrani, ale pri generovani postavy sa to nepouziva a je zbytocne to zobrazovat

Reply:
no hej ale potom to bude nekonzistentne ked to bude skakat a bude matuce pre hraca

Reply:
asi nerozumiem co bude skakat. Na obrazovke je treba zobrazovat len veci, ktore prave potrebujes, aby si sa v tom neztracal. 
A pri generovani postavy hunger nepotrebujes

Reply:
chcem povedat ze ano, na generovanie postavy nema hunger vpliv
je to statistika o ktorej by hrac mohol byt informovany, lebo je to cast ktoru by si mozno chcel tematicky menit.
s tymto nesuhlasis ale to je ok
no ja este hovorim ze informacne okno s zivotnymi statami by malo byt konzistentne bez ohladu na cast applikacie (generacia char, edit/read char, relations map apod)

Reply:
co ja viem, ci malo. Ja si stale predstavujem, ze vysledok je ten papierovy denik, ktory si vytlacim :-) ale tak svoj nazor som ti povedal aj zdovodnil. Ak na tom stale trvas, ze tam chces Hunger, tak ustupim